researchapp/templates/consent.jinja2

```diff
@@ -6,20 +6,18 @@
 
 {% block body %}
 <div>
-    <form action="http://52.39.26.206:9000/authorize/" id="authorize-form">
-        <input type="hidden" name="client_id" value="{{ oauth_config.client_id }}" />
+    <form action="{{ provider.authorize_url }}" id="authorize-form">
+        <input type="hidden" name="client_id" value="app-demo" />
         <input type="hidden" name="response_type" value="code" />
         <input type="hidden" name="scope" value="patient/*.read offline_access" />
         <input type="hidden" name="redirect_uri" value="http://52.39.26.206:6543/authorized" />
         <input type="hidden" name="state" value="123" />
```

Comment:
Eventually this needs to be a high entropy random value that the server component stores (and verifies after redirection  back to the app).